test/restart/common.go

```diff
@@ -0,0 +1,30 @@
+package restart
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
+)
+
+func LogCheck(cmd string) string {
```

Comment:
nit - returning a string is odd to me here, especially line 26. IS this supposed to be an error or is it supposed to just be some status?